sanic/config.py

```diff
@@ -222,6 +222,12 @@ def __init__(self, name) -> None:
         for key, value in environ.items():
             if not key.startswith(prefix):
                 continue
+            if not key.isupper():
+                deprecation(
+                    "Lowercase environment variables won't be "
+                    "catch for sanic config in the future version, 22.9",
```

Comment:
Might want to use the word `loaded into` instead of `catch for` to stay consistent with the documentation and avoid misunderstandings, as `catch` is used for exception handling in other languages.

And a small nitpick: uppercase in “Sanic”, please 😅

Reply:
Thanks! Fixed.

Reply:
Thanks! :)